lib/config/options/index.ts

```diff
@@ -484,14 +484,14 @@ const options: RenovateOptions[] = [
   // Log options
   {
     name: 'logFile',
-    description: 'Log file path.',
+    description: 'Log file path. (deprecated)',
```

Comment:
We should add a `deprecated` boolean field, which automatically render a warning to the docs, like we do for `experimental`

https://github.com/renovatebot/renovate/blob/a6f90f26698d21db64f2732cb114566a4d3d1d92/tools/docs/config.ts#L157

Reply:
https://github.com/renovatebot/renovate/pull/28216